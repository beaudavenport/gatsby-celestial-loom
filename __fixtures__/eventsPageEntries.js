export default {
  allMarkdownRemark: {
    totalCount: 2,
    edges: [{
      node: {
        id: 'f0ecfd4d-1054-5fa2-90eb-5e9bff25e985',
        frontmatter: {
          title: 'Moon Circle',
          image: '/assets/gemini-moon-circle.jpg',
          eventDate: 'May 30, 2019',
          eventTime: '7 PM - 9 PM',
          eventDateShort: 'May 30',
          eventPrice: 25,
          priceDescription: '2 Hour Playshop',
          location: 'Divine Inspirations',
        },
        excerpt: '<p>Join us for a Goddess GEMINI Moon (12 Gemini) celebration at our sacred MOON CIRCLE.  We will explore the making "connections" lunar unfolding in June, learn about the "Lights Mandala" and apply our personal chart Lunar Planner to our monthly timing.</p>',
        fields: { slug: '/events/moon-circle/' },
      },
    }, {
      node: {
        id: 'b3ffaeb7-2067-54b0-a015-df104b6e1e34',
        frontmatter: {
          title: 'STONES OF FIRE',
          image: '/assets/stones-of-fire-lotus.jpg',
          eventDate: 'Jun 08, 2019',
          eventTime: '1PM - 4 PM',
          eventDateShort: 'Jun 08',
          eventPrice: 35,
          priceDescription: '3 Hour Playshop',
          location: 'DeMun Healing',
        },
        excerpt: '<p>A Healing Playshop</p>\n<p>Saturday, June 8, 2019, 1-4 PM</p>\n<p>DeMun Healing, 734 De Mun Ave., Clayton, MO 63105</p>\n<p>Every action, thought and emotion is empowered by a corresponding unending flow of energy – the Spinal Column is the <em>Trunk of Life</em> in the <em>Tree of Man</em>.  The chakras are the <strong>Stones of Fire</strong>…</p>',
        fields: { slug: '/events/stones-of-fire/' },
      },
    }],
  },
};
