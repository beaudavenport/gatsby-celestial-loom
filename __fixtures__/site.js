export default {
  site: {
    siteMetadata: {
      title: 'The Celestial Loom',
    },
  },
};
