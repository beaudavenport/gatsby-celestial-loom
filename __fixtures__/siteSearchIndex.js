export default {
  siteSearchIndex: {
    index: null,
  },
};
