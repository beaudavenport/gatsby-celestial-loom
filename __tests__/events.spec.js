import { StaticQuery } from 'gatsby';
import React from 'react';

import renderer from 'react-test-renderer';

import Events from '../src/pages/events';
import eventArchiveEntries from '../__fixtures__/eventArchiveEntries';
import eventsPageEntries from '../__fixtures__/eventsPageEntries';
import sidebarEntries from '../__fixtures__/sidebarEntries';
import site from '../__fixtures__/site';
import siteSearchIndex from '../__fixtures__/siteSearchIndex';

beforeEach(() => {
  global.window.matchMedia = jest.fn().mockImplementation(query => ({
    matches: false,
    media: query,
    onchange: null,
    addListener: jest.fn(),
    removeListener: jest.fn(),
  }));

  StaticQuery.mockImplementation(({ render }) => render({
    ...site,
    ...siteSearchIndex,
    ...eventArchiveEntries,
    ...sidebarEntries,
  }));
});

describe('Events', () => {
  it('should render correctly with events', () => {
    const tree = renderer
      .create(<Events data={eventsPageEntries} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
