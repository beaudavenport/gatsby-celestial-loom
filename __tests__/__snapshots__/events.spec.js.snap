// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Events should render correctly with events 1`] = `
<div>
  <header
    className="md-paper md-paper--2 md-toolbar md-background--primary md-toolbar--text-white md-toolbar--fixed"
    style={
      Object {
        "backgroundColor": "rgb(47, 47, 47)",
      }
    }
  >
    <button
      className="md-btn md-btn--icon md-pointer--hover md-inline-block md-btn--toolbar md-toolbar--action-left"
      disabled={false}
      onClick={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      type="button"
    >
      <div
        className="md-ink-container"
      />
      <i
        className="md-icon material-icons md-text--inherit"
      >
        menu
      </i>
    </button>
    <h2
      className="md-title md-title--toolbar"
      style={
        Object {
          "fontFamily": "Berkshire Swash",
          "fontStyle": "italic",
        }
      }
    >
      Events
    </h2>
    <div
      className="md-cell--right md-toolbar--action-right"
    >
      <div
        className="md-btn--toolbar header-icons-container"
      >
        <div
          className="header-search-container"
        >
          <button
            className="md-btn md-pointer--hover md-text md-inline-block"
            onClick={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchStart={[Function]}
            style={
              Object {
                "alignItems": "center",
                "display": "flex",
              }
            }
            type="button"
          >
            <div
              className="md-ink-container"
            />
            <div
              className="md-badge-container md-inline-block"
            >
              <i
                className="md-icon material-icons"
                style={
                  Object {
                    "fontSize": "2rem",
                  }
                }
              >
                shopping_cart
              </i>
            </div>
          </button>
        </div>
        <div
          className="header-search-container"
        >
          <button
            className="md-btn md-pointer--hover md-text md-inline-block"
            onClick={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchStart={[Function]}
            style={
              Object {
                "alignItems": "center",
                "display": "flex",
              }
            }
            type="button"
          >
            <div
              className="md-ink-container"
            />
            <i
              className="md-icon material-icons"
              style={
                Object {
                  "fontSize": "2rem",
                }
              }
            >
              search
            </i>
          </button>
        </div>
      </div>
    </div>
  </header>
  <main
    className="md-navigation-drawer-content md-navigation-drawer-content--inactive md-transition--acceleration md-toolbar-relative"
    id="main-content"
    tabIndex={-1}
  >
    <div
      className="md-grid md-grid--no-spacing"
      style={
        Object {
          "maxWidth": 1100,
        }
      }
    >
      <div
        className="md-cell md-cell--8 main-contents"
      >
        <div
          className="link--container false"
        >
          <a
            href="/"
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <div
              className="touchable-link "
              style={Object {}}
            >
              <div
                className="md-ink-container"
              />
              <div
                className="flex-center"
              >
                <i
                  className="md-icon material-icons"
                  style={
                    Object {
                      "color": "#ec6602",
                      "fontSize": "2rem",
                      "marginRight": 5,
                    }
                  }
                >
                  arrow_back
                </i>
                <p
                  className="back-title"
                >
                  Home
                </p>
              </div>
            </div>
          </a>
        </div>
        <div
          className="md-grid"
        >
          <div
            className="md-cell md-cell--12"
          >
            <div
              style={
                Object {
                  "marginBottom": 30,
                }
              }
            >
              <p
                className="subheader"
                style={
                  Object {
                    "fontSize": "2rem",
                    "lineHeight": "2.5rem",
                  }
                }
              >
                Join us in-person!
              </p>
              <p
                style={
                  Object {
                    "fontWeight": "bold",
                  }
                }
              >
                Nikki provides playshops, speaking engagements, and more around the St. Louis area.
              </p>
            </div>
          </div>
          <p
            className="subheader"
            style={
              Object {
                "fontSize": "1.5rem",
                "lineHeight": "1.5rem",
              }
            }
          >
            Upcoming Events
          </p>
          <div
            className="md-cell md-cell--12"
            style={
              Object {
                "padding": "20px 0px",
              }
            }
          >
            <a
              href="/events/moon-circle/"
              style={
                Object {
                  "textDecoration": "none",
                }
              }
            >
              <div
                className="touchable-link "
                style={Object {}}
              >
                <div
                  className="md-ink-container"
                />
                <div
                  className="md-grid"
                >
                  <div
                    className="md-cell md-cell--6 md-cell--4-tablet"
                  >
                    <div
                      className="image-container"
                    >
                      <img
                        alt="Preview of featured post"
                        className="image-container--image"
                        src="/assets/gemini-moon-circle.jpg"
                      />
                      <div
                        style={
                          Object {
                            "alignItems": "middle",
                            "backgroundColor": "rgb(247, 105, 0)",
                            "display": "flex",
                            "flexDirection": "column",
                            "left": 0,
                            "marginBottom": 10,
                            "padding": "5px 10px",
                            "position": "absolute",
                            "top": 0,
                          }
                        }
                      >
                        <i
                          className="md-icon material-icons"
                          style={
                            Object {
                              "color": "white",
                            }
                          }
                        >
                          event
                        </i>
                        <p
                          style={
                            Object {
                              "color": "white",
                              "fontStyle": "italic",
                              "fontWeight": "bold",
                              "lineHeight": "24px",
                              "marginBottom": 0,
                              "marginLeft": 3,
                            }
                          }
                        >
                          May 30
                        </p>
                      </div>
                    </div>
                  </div>
                  <div
                    className="md-cell md-cell--6 md-cell--4-tablet"
                  >
                    <div
                      style={
                        Object {
                          "padding": 16,
                        }
                      }
                    >
                      <p
                        className="subheader"
                      >
                        Upcoming Event
                      </p>
                      <h2
                        className="title"
                      >
                        Moon Circle OF THE FUTURE
                      </h2>
                    </div>
                    <div
                      className="event-detail-container"
                    >
                      <div
                        className="content-with-icon"
                      >
                        <i
                          className="md-icon material-icons"
                          style={
                            Object {
                              "marginRight": 5,
                            }
                          }
                        >
                          place
                        </i>
                        <p
                          className="info-text"
                        >
                          Divine Inspirations
                        </p>
                      </div>
                      <div
                        className="content-with-icon"
                      >
                        <i
                          className="md-icon material-icons"
                          style={
                            Object {
                              "marginRight": 5,
                            }
                          }
                        >
                          event
                        </i>
                        <p
                          className="info-text"
                        >
                          May 30, 2119 - 7 PM - 9 PM
                        </p>
                      </div>
                    </div>
                    <div
                      className="event-signup-thumbnail"
                    >
                      <div>
                        <p
                          className="event-signup-thumbnail--price-description"
                        >
                          2 Hour Playshop
                        </p>
                        <p
                          className="event-signup-thumbnail--price"
                        >
                          $25.00
                        </p>
                      </div>
                      <div>
                        <div
                          className="chevron-text"
                        >
                          <p
                            className="chevron-text--text"
                          >
                            Sign Up
                          </p>
                          <i
                            className="md-icon material-icons chevron-text--chevron"
                          >
                            chevron_right
                          </i>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </a>
          </div>
          <div
            className="md-cell md-cell--12"
            style={
              Object {
                "padding": "20px 0px",
              }
            }
          >
            <a
              href="/events/stones-of-fire/"
              style={
                Object {
                  "textDecoration": "none",
                }
              }
            >
              <div
                className="touchable-link "
                style={Object {}}
              >
                <div
                  className="md-ink-container"
                />
                <div
                  className="md-grid"
                >
                  <div
                    className="md-cell md-cell--6 md-cell--4-tablet"
                  >
                    <div
                      className="image-container"
                    >
                      <img
                        alt="Preview of featured post"
                        className="image-container--image"
                        src="/assets/stones-of-fire-lotus.jpg"
                      />
                      <div
                        style={
                          Object {
                            "alignItems": "middle",
                            "backgroundColor": "rgb(247, 105, 0)",
                            "display": "flex",
                            "flexDirection": "column",
                            "left": 0,
                            "marginBottom": 10,
                            "padding": "5px 10px",
                            "position": "absolute",
                            "top": 0,
                          }
                        }
                      >
                        <i
                          className="md-icon material-icons"
                          style={
                            Object {
                              "color": "white",
                            }
                          }
                        >
                          event
                        </i>
                        <p
                          style={
                            Object {
                              "color": "white",
                              "fontStyle": "italic",
                              "fontWeight": "bold",
                              "lineHeight": "24px",
                              "marginBottom": 0,
                              "marginLeft": 3,
                            }
                          }
                        >
                          Jun 08
                        </p>
                      </div>
                    </div>
                  </div>
                  <div
                    className="md-cell md-cell--6 md-cell--4-tablet"
                  >
                    <div
                      style={
                        Object {
                          "padding": 16,
                        }
                      }
                    >
                      <p
                        className="subheader"
                      >
                        Upcoming Event
                      </p>
                      <h2
                        className="title"
                      >
                        STONES OF FIRE OF THE FUTURE
                      </h2>
                    </div>
                    <div
                      className="event-detail-container"
                    >
                      <div
                        className="content-with-icon"
                      >
                        <i
                          className="md-icon material-icons"
                          style={
                            Object {
                              "marginRight": 5,
                            }
                          }
                        >
                          place
                        </i>
                        <p
                          className="info-text"
                        >
                          DeMun Healing
                        </p>
                      </div>
                      <div
                        className="content-with-icon"
                      >
                        <i
                          className="md-icon material-icons"
                          style={
                            Object {
                              "marginRight": 5,
                            }
                          }
                        >
                          event
                        </i>
                        <p
                          className="info-text"
                        >
                          Jun 08, 2119 - 1PM - 4 PM
                        </p>
                      </div>
                    </div>
                    <div
                      className="event-signup-thumbnail"
                    >
                      <div>
                        <p
                          className="event-signup-thumbnail--price-description"
                        >
                          3 Hour Playshop
                        </p>
                        <p
                          className="event-signup-thumbnail--price"
                        >
                          $35.00
                        </p>
                      </div>
                      <div>
                        <div
                          className="chevron-text"
                        >
                          <p
                            className="chevron-text--text"
                          >
                            Sign Up
                          </p>
                          <i
                            className="md-icon material-icons chevron-text--chevron"
                          >
                            chevron_right
                          </i>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </a>
          </div>
          <p
            className="subheader"
            style={
              Object {
                "fontSize": "1.5rem",
                "lineHeight": "1.5rem",
              }
            }
          >
            Past Events
          </p>
        </div>
      </div>
      <div
        className="md-cell md-cell--4 md-cell--8-tablet sidebar-contents"
      >
        <div
          className="archive--container"
        >
          <div
            className="sidebar-header"
          >
            <h5
              className="sidebar-header--title"
            >
              Events Calendar
            </h5>
          </div>
          <div>
            <ul
              className="md-list"
            >
              <div>
                <li
                  className="md-list-item md-list-item--nested-container"
                  style={Object {}}
                >
                  <div
                    aria-expanded={false}
                    aria-pressed={false}
                    className="md-fake-btn md-pointer--hover md-fake-btn--no-outline md-list-tile md-list-tile--two-lines md-text"
                    onBlur={[Function]}
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    onKeyUp={[Function]}
                    onMouseLeave={[Function]}
                    onMouseOver={[Function]}
                    onTouchEnd={[Function]}
                    onTouchStart={[Function]}
                    role="button"
                    tabIndex={0}
                  >
                    <div
                      className="md-ink-container"
                    />
                    <div
                      className="md-tile-content"
                    >
                      <div
                        className="md-tile-text--primary md-text"
                        style={
                          Object {
                            "fontWeight": "bold",
                          }
                        }
                      >
                        May 2019
                      </div>
                      <div
                        className="md-tile-text--secondary md-text--secondary"
                        style={
                          Object {
                            "fontWeight": "bold",
                          }
                        }
                      >
                        1 events
                      </div>
                    </div>
                    <div
                      className="md-tile-addon md-tile-addon--icon"
                    >
                      <i
                        className="md-icon material-icons md-collapser"
                      >
                        keyboard_arrow_down
                      </i>
                    </div>
                  </div>
                </li>
              </div>
              <div>
                <li
                  className="md-list-item md-list-item--nested-container"
                  style={Object {}}
                >
                  <div
                    aria-expanded={false}
                    aria-pressed={false}
                    className="md-fake-btn md-pointer--hover md-fake-btn--no-outline md-list-tile md-list-tile--two-lines md-text"
                    onBlur={[Function]}
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    onKeyUp={[Function]}
                    onMouseLeave={[Function]}
                    onMouseOver={[Function]}
                    onTouchEnd={[Function]}
                    onTouchStart={[Function]}
                    role="button"
                    tabIndex={0}
                  >
                    <div
                      className="md-ink-container"
                    />
                    <div
                      className="md-tile-content"
                    >
                      <div
                        className="md-tile-text--primary md-text"
                        style={
                          Object {
                            "fontWeight": "bold",
                          }
                        }
                      >
                        June 2019
                      </div>
                      <div
                        className="md-tile-text--secondary md-text--secondary"
                        style={
                          Object {
                            "fontWeight": "bold",
                          }
                        }
                      >
                        1 events
                      </div>
                    </div>
                    <div
                      className="md-tile-addon md-tile-addon--icon"
                    >
                      <i
                        className="md-icon material-icons md-collapser"
                      >
                        keyboard_arrow_down
                      </i>
                    </div>
                  </div>
                </li>
              </div>
            </ul>
          </div>
        </div>
        <div>
          <div
            className="sidebar-header"
          >
            <h5
              className="sidebar-header--title"
            >
              Latest Updates
            </h5>
          </div>
          <div
            style={
              Object {
                "padding": 10,
              }
            }
          >
            <a
              href="/blog/norwac-i-am-on-my-way/"
              style={
                Object {
                  "textDecoration": "none",
                }
              }
            >
              <div
                className="touchable-link thumbnail-card"
                style={Object {}}
              >
                <div
                  className="md-ink-container"
                />
                <div
                  className="thumbnail-image"
                  style={
                    Object {
                      "backgroundImage": "url(/assets/cancer-norwac.jpg)",
                    }
                  }
                />
                <div
                  className="thumbnail-text"
                >
                  <p
                    className="thumbnail-text--title"
                  >
                    NORWAC - I AM ON MY WAY
                  </p>
                  <p
                    className="thumbnail-text--subtitle"
                  >
                    Posted on:
                     
                    <span
                      style={
                        Object {
                          "color": "rgba(0, 0, 0, 0.87)",
                          "fontWeight": "bold",
                        }
                      }
                    >
                      May 23, 2019
                    </span>
                  </p>
                </div>
              </div>
            </a>
            <a
              href="/blog/the-pleiades/"
              style={
                Object {
                  "textDecoration": "none",
                }
              }
            >
              <div
                className="touchable-link thumbnail-card"
                style={Object {}}
              >
                <div
                  className="md-ink-container"
                />
                <div
                  className="thumbnail-image"
                  style={
                    Object {
                      "backgroundImage": "url(/assets/pleiades.jpg)",
                    }
                  }
                />
                <div
                  className="thumbnail-text"
                >
                  <p
                    className="thumbnail-text--title"
                  >
                    The Pleiades
                  </p>
                  <p
                    className="thumbnail-text--subtitle"
                  >
                    Posted on:
                     
                    <span
                      style={
                        Object {
                          "color": "rgba(0, 0, 0, 0.87)",
                          "fontWeight": "bold",
                        }
                      }
                    >
                      May 21, 2019
                    </span>
                  </p>
                </div>
              </div>
            </a>
          </div>
        </div>
      </div>
    </div>
    <div
      className="md-grid footer--grid"
    >
      <div
        className="md-cell md-cell--6 md-cell--8-tablet"
      >
        <div
          className="flex-center"
        >
          <div
            style={
              Object {
                "marginRight": 10,
              }
            }
          >
            <img
              alt="Celestial Loom Logo"
              src="File contents (usually images)"
              style={
                Object {
                  "width": 50,
                }
              }
            />
          </div>
          <h3
            className="footer--title"
          >
            The Celestial Loom
          </h3>
        </div>
        <div
          style={
            Object {
              "padding": 10,
            }
          }
        >
          <p
            className="footer--text"
          >
            ©
             Copyright 2019, Nikki Davenport.
          </p>
          <p
            className="footer--text"
          >
            Cover photography by Photo by Anastasia Dulgier on Unsplash.
          </p>
          <p
            className="footer--text"
          >
            Built with Gatsby and Netflify CMS.
          </p>
        </div>
      </div>
      <div
        className="md-cell md-cell--6 md-cell--8-tablet"
      >
        <div
          className="md-grid"
        >
          <div
            className="md-cell md-cell--6 md-cell--4-tablet"
          >
            <p
              className="subheader"
            >
              Links
            </p>
            <ul
              className="footer--list"
            >
              <li
                className="footer--line-item"
              >
                <a
                  href="/home/"
                  style={
                    Object {
                      "color": "white",
                    }
                  }
                >
                  Home
                </a>
              </li>
              <li
                className="footer--line-item"
              >
                <a
                  href="/about-nikki/"
                  style={
                    Object {
                      "color": "white",
                    }
                  }
                >
                  About Nikki
                </a>
              </li>
              <li
                className="footer--line-item"
              >
                <a
                  href="/getting-started/"
                  style={
                    Object {
                      "color": "white",
                    }
                  }
                >
                  Getting Started
                </a>
              </li>
              <li
                className="footer--line-item"
              >
                <a
                  href="/posts/"
                  style={
                    Object {
                      "color": "white",
                    }
                  }
                >
                  Posts
                </a>
              </li>
              <li
                className="footer--line-item"
              >
                <a
                  href="/events/"
                  style={
                    Object {
                      "color": "white",
                    }
                  }
                >
                  Events
                </a>
              </li>
              <li
                className="footer--line-item"
              >
                <a
                  href="/services/"
                  style={
                    Object {
                      "color": "white",
                    }
                  }
                >
                  Services
                </a>
              </li>
              <li
                className="footer--line-item"
              >
                <a
                  href="/toolbox/"
                  style={
                    Object {
                      "color": "white",
                    }
                  }
                >
                  Astro Toolbox
                </a>
              </li>
            </ul>
          </div>
          <div
            className="md-cell md-cell--6 md-cell--4-tablet"
          >
            <p
              className="subheader"
            >
              Contact
            </p>
            <ul
              className="footer--list"
            >
              <li>
                <div
                  className="footer--link"
                >
                  <i
                    className="md-icon material-icons"
                    style={
                      Object {
                        "color": "white",
                      }
                    }
                  >
                    email
                  </i>
                  <a
                    className="footer--link__link"
                    href="mailto: nikiastro@att.net"
                  >
                    Email Nikki
                  </a>
                </div>
              </li>
              <li>
                <div
                  className="footer--link"
                >
                  <svg
                    fill="white"
                    height={20}
                    viewBox="0 0 24 24"
                    width={20}
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M12 0c-6.627 0-12 5.373-12 12s5.373 12 12 12 12-5.373 12-12-5.373-12-12-12zm3 8h-1.35c-.538 0-.65.221-.65.778v1.222h2l-.209 2h-1.791v7h-3v-7h-2v-2h2v-2.308c0-1.769.931-2.692 3.029-2.692h1.971v3z"
                    />
                  </svg>
                  <a
                    className="footer--link__link"
                    href="https://www.facebook.com/celestialloom/"
                  >
                    Facebook
                  </a>
                </div>
              </li>
              <li>
                <div
                  className="footer--link"
                >
                  <svg
                    fill="white"
                    height={20}
                    viewBox="0 0 24 24"
                    width={20}
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M12 0c-6.627 0-12 5.373-12 12s5.373 12 12 12 12-5.373 12-12-5.373-12-12-12zm6.066 9.645c.183 4.04-2.83 8.544-8.164 8.544-1.622 0-3.131-.476-4.402-1.291 1.524.18 3.045-.244 4.252-1.189-1.256-.023-2.317-.854-2.684-1.995.451.086.895.061 1.298-.049-1.381-.278-2.335-1.522-2.304-2.853.388.215.83.344 1.301.359-1.279-.855-1.641-2.544-.889-3.835 1.416 1.738 3.533 2.881 5.92 3.001-.419-1.796.944-3.527 2.799-3.527.825 0 1.572.349 2.096.907.654-.128 1.27-.368 1.824-.697-.215.671-.67 1.233-1.263 1.589.581-.07 1.135-.224 1.649-.453-.384.578-.87 1.084-1.433 1.489z"
                    />
                  </svg>
                  <a
                    className="footer--link__link"
                    href="https://twitter.com/nikiastro"
                  >
                    Twitter
                  </a>
                </div>
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
  </main>
</div>
`;
